packages/docs/docs/contributing/run-the-stack.md

```diff
@@ -144,6 +144,12 @@ If everything is working, you should see the following in your browser:
 { "ok": true, "version": "<MEDPLUM_VERSION>", "postgres": true, "redis": true }
 ```
 
+:::warning Troubleshooting
+
+If you get an error such as `error: role "admin@example.com" does not exist` or `password authentication failed for user "medplum"` this usually means you have another instance of Postgres running. Closing the other instance should resolve this issue.
```

Comment:
I think this should be `role "medplum" does not exist`

"admin@example.com" is in our application layer.  The database username is typically "medplum".